samples/subsys/fs/fat_fs/boards/nucleo_l4r5zi.overlay

```diff
@@ -0,0 +1,6 @@
+&sdmmc1 {
```

Comment:
I might have been too quick in approving this.

If I'm correct, there is no SDMMC peripheral on this board. 
This won't work for users unless they add the component, which at least requires some comments in sample REAMDE or directly in this file.
Interest of this modification is that it allows to see that SDMMC driver compiles on L4+ target, but this would require some additional modifications on sample, so this could be run in CI with command:
` twister -b -c -T samples/subsys/fs/fat_fs/ -p nucleo_l4r5zi`